src/TransactionTestsFiller/ttEIP1559/maxPriorityFeePerGas00prefixPriorityFeeGreaterThanBaseFeeFiller.json

```diff
@@ -0,0 +1,24 @@
+{
+    "maxPriorityFeePerGas00prefixPriorityFeeGreaterThanBaseFee" :
+    {
+	    "expectException" :
+        {
+          "<=Berlin" : "TR_TypeNotSupported",
+          ">=London" : "1559PriorityFeeGreaterThanBaseFee"
+        },
+        "transaction" :
+        {
+            "data" : "",
+            "gasLimit" : "21000",
+            "maxFeePerGas" : "0x00fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe",
+            "maxPriorityFeePerGas" : "0x00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
```

Comment:
I don't think that 00 is preserved in rlp here. 
Whats the transaction rlp build for t9n?

Reply:
Updated to have "0x:bigint". Fails as expected with exception "1559LeadingZerosPriorityFee". Separated max gas value test in new file.